io.gitlab.android_translation_layer.BaseApp.yml

```diff
@@ -0,0 +1,275 @@
+      - type: git
+        dest: third_party/externals/spirv-headers
+        url: https://github.com/KhronosGroup/spirv-headers.git
+        commit: bdbfd019be6952fd8fa9bd5606a8798a7530c853
+      - type: git
+        dest: third_party/externals/spirv-tools
+        url: https://github.com/KhronosGroup/spirv-tools.git
+        commit: e7c6084fd1d6d6f5ac393e842728d8be309688ca
+      - type: git
+        dest: third_party/externals/vello
+        url: https://github.com/linebender/vello.git
+        commit: ef2630ad9c647b90863cb0915701d54725733968
+      - type: git
+        dest: third_party/externals/vulkan-headers
+        url: https://github.com/KhronosGroup/vulkan-headers.git
+        commit: 9e61870ecbd32514113b467e0a0c46f60ed222c7
+      - type: git
+        dest: third_party/externals/vulkan-tools
+        url: https://github.com/KhronosGroup/vulkan-tools.git
+        commit: 80b010b1e1b08c1f092fb2bfa337faadf8ea1ba3
+      - type: git
+        dest: third_party/externals/wuffs
+        url: https://github.com/google/wuffs-mirror-release-c.git
+        commit: a0041ac0310b3156b963e2f2bea09245f25ec073
+
+  - name: wolfssl
+    buildsystem: autotools
+    config-opts:
+      - --enable-shared
+      - --disable-opensslall
+      - --disable-opensslextra
+      - --enable-aescbc-length-checks
+      - --enable-curve25519
+      - --enable-ed25519
+      - --enable-ed25519-stream
+      - --enable-oldtls
+      - --enable-base64encode
+      - --enable-tlsx
+      - --enable-scrypt
+      - --disable-examples
+      - --enable-crl
+      - --with-rsa
+      - --enable-certs
+      - --enable-session-certs
+      - --enable-encrypted-keys
+      - --enable-cert-gen
+      - --enable-cert-ext
+      - --enable-clr-monitor
+      - --enable-jni
+    sources:
+      - type: git
+        url: https://github.com/wolfSSL/wolfssl.git
+        tag: v5.7.0-stable
+
+  - name: vixl
+    buildsystem: meson
+    build-options:
+      config-opts:
+        - -Dsimulator=none
+    only-arches:
+      - aarch64
+    sources:
+      - type: archive
+        url: https://github.com/Linaro/vixl/archive/refs/tags/7.0.0.tar.gz
+        sha256: 2abe97bfec41995e6839051951beac9fc57ef6f1a2ae10818bf2842cdc3b1de1
+      - type: patch
+        path: patches/vixl_meson_support.patch
+
+  - name: bionic_translation
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://gitlab.com/android_translation_layer/bionic_translation.git
+        commit: 8225d8a502aa4f94cfdb5a0e67eee0c84ddba2ac
+      - type: patch
+        path: patches/bionic_translation.patch
+
+  - name: art_standalone
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/openjdk17/bin
+      env:
+        JAVA_HOME: /usr/lib/sdk/openjdk17/jvm/openjdk-17
+        LIBRARY_PATH: /app/lib
+    build-commands:
+      - make -j8
+      - make ____PREFIX=/app ____LIBDIR=lib ____INSTALL_ETC=/app/etc install
+    sources:
```

Comment:
use `no-autogen: true` and `make-install-args` with the options, drop `-j8`, build-commands and `buildsystem: simple`

Reply:
done